.github/cspell.json

```diff
@@ -14,8 +14,21 @@
         "pyproject",
         "virtualenvs",
         "git",
+        "docstrings",
+        "Usefullness",
```

Comment:
This is not the correct spelling of "usefulness" -- it should probably not be marked as an exception for cspell.

Reply:
```suggestion
```

Reply:
@solisa986